mods/default/nodes.lua

```diff
@@ -2077,6 +2077,7 @@ minetest.register_node("default:coral_green", {
 	tiles = {"default_coral_skeleton.png"},
 	special_tiles = {{name = "default_coral_green.png", tileable_vertical = true}},
 	inventory_image = "default_coral_green.png",
+    wield_image = "default_coral_green.png",
```

Comment:
tabs, not spaces

Reply:
Hmm, that's odd. I thought my editor was set correctly...

Reply:
Should be fixed now.